src/index.html

```diff
@@ -11,6 +11,37 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <!-- <h1>Search bar airbnb</h1> -->
+    <div class="search-bar-big">
+      <div class="search-container-big">
+        <form id="big" 
+        action="https://localhost:3000/api" 
+        method="post" >
```

Comment:
```suggestion
        <form 
          id="big" 
          action="https://localhost:3000/api" 
          method="post" 
        >
```

Reply:
check it everywhere and keep the code style

Reply:
fixed that, thank you